### DISCUSSION TITLE
Description from docstring in the Swagger UI

### DISCUSSION THREAD

**Discussion by harshalizode** [dymmond/esmerald#397] [Q&A] [answered]
Hello Everyone,

As the FastAPI we have a feature to add the description from doc-string in the Swagger UI?

**Comment by tarsil** [comment]
You also have in Esmerald, when you do add the handler like a get or a post or whatever, you can pass a description there. Example:

@post("/create", description=".....")

This is also in the docs

**Comment by tarsil** [comment]
> Hello Everyone,
> 
> As the FastAPI we have a feature to add the description from doc-string in the Swagger UI?

FastAPI?

**Reply by harshalizode** [reply]
Here , is the example [](https://fastapi.tiangolo.com/tutorial/path-operation-configuration/?h=docstring#description-from-docstring)

To add the description in the swagger ui we need to add it in handlers decorator, but if we not provide then it should be generate the description from the doc-string.

**Reply by tarsil** [reply]
Hmm, that is actually a very good point. The current version is not doing that and we are preparing something for the next release. We will add your suggestion there. Good point @harshalizode

**Reply by harshalizode** [reply]
Could you please provide an update on the timeline for the next release of Esmerald? It would help us plan accordingly.

**Reply by tarsil** [reply]
We are aiming for the end of next week maximum.

**Reply by tarsil** [reply]
> Could you please provide an update on the timeline for the next release of Esmerald? It would help us plan accordingly.

@harshalizode to give you an idea, its already here https://github.com/dymmond/esmerald/pull/398

So, did you replace your FastAPI application with Esmerald or you have both working side by side?

**Reply by harshalizode** [reply]
No, we are using the Esmerald only in our project.

**Reply by tarsil** [reply]
Ok, thank you for that :D We will try to release as soon as possible 👍🏼

**Reply by harshalizode** [reply]
@tarsil Can you check the issue in mongoz [https://github.com/dymmond/mongoz/issues/61](url)
I need some advice on it.

**Reply by tarsil** [reply]
> re aiming for the end of next week maximum.

We received the email but I'm currently doing some work but I will have a look today @harshalizode

**Reply by harshalizode** [reply]
Okay, Thanks.

**Reply by tarsil** [reply]
> Okay, Thanks.

Answered there and released 👍🏼

**Reply by harshalizode** [reply]
Yes, thank for the response :D

**Reply by tarsil** [reply]
@harshalizode release 3.4.1 done today and this feature also released for the swagger. What you mentioned.

**Reply by harshalizode** [reply]
@tarsil, thanks for the update.

**Reply by harshalizode** [reply, marked as answer]
yes, is it working as expected.
Thanks for the release.
